ember_debug/adapters/basic.js

```diff
@@ -113,7 +113,7 @@ export default EmberObject.extend({
   */
   connect() {
     return new Promise((resolve, reject) => {
-      $(() => {
+      document.addEventListener('DOMContentLoaded', () => {
```

Comment:
also what browsers does this target? a few IE versions have a bug where an `interactive` `document.readyState` that's not actually mean it's ready to access and manipulate